lib/Advisor.js

```diff
@@ -164,7 +165,7 @@ class Advisor {
       const button = this.getButton(win);
       button.hide();
       simpleStorage.storage.recData = undefined; // clear all recs
-      this.uninstallAddon(self.id, '');
+      this.uninstallAddon(sdkData.id, '');
```

Comment:
@casebenton Can you r? this change? Seems like this would fail currently, but I want a second pair of eyes since I just sort've guessed at the fix.

Also, weirdly, when I test locally, current master doesn't hit an error on this line, even though:

1. `self` is undefined in the file itself.
2. `self` within content scripts is a different `self` from the Add-on SDK module, and, [according to the docs](https://developer.mozilla.org/en-US/Add-ons/SDK/Guides/Content_Scripts/self), doesn't have an `id` property.

What's the deal here?